xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -181,11 +188,15 @@ void CGUIWindowPVRBase::OnInitWindow(void)
 
   // mark item as selected by channel path
   m_viewControl.SetSelectedItem(GetSelectedItemPath(m_bRadio));
+
+  RegisterObservers();
 }
 
 void CGUIWindowPVRBase::OnDeinitWindow(int nextWindowID)
 {
+  UnregisterObservers();
   UpdateSelectedItemPath();
+  CGUIWindowPVRBase::OnDeinitWindow(nextWindowID);
```

Comment:
@FernetMenta Ofc. Sorry, already fixed. You were just to fast. On my way to work while exchanging emails with you I realized that base class never was called here and I "fixed" this on the fly - with the result you noticed... One never should commit stuff without runtime testing. Shame on me. ;-)